src/stylesheets/components/_footer.scss

```diff
@@ -191,31 +191,37 @@ li.usa-footer-primary-content {
       background-position: 1.5rem center;
       background-repeat: no-repeat;
       background-size: 1.3rem;
+      margin: 0;
       padding-left: 3.5rem;
 
       @include media($medium-screen) {
         background: none;
+        margin-bottom: .8rem;
         padding-bottom: 0;
         padding-left: 0;
       }
     }
 
+    h4 {
+      margin: 0;
+    }
```

Comment:
What if they use a different heading, should we allow this or keep it strictly h4?